packages/arb-token-bridge-ui/public/token-list-42161.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Arb Whitelist Era",
```

Comment:
If we're generating the token list in `arb-token-lists` is the intended flow to then copy and paste it here?

I'm assuming we have this here so others can query `https://bridge.arbitrum.io/token-list-42161.json`
We could instead surface this some other way, or just have a redirect to `https://tokenlist.arbitrum.io/...`